src/sentry/integrations/msteams/__init__.py

```diff
@@ -3,7 +3,11 @@
 from .actions.form import *  # noqa: F401,F403
 from .actions.notification import *  # noqa: F401,F403
 from .actions.notification import MsTeamsNotifyServiceAction
-from .card_builder import *  # noqa: F401,F403
```

Comment:
There's a utils module in the parent module as well, one of patch calls in a test gets confused because of this, so import everything except utils.